yarn.lock

```diff
@@ -356,6 +377,10 @@ ansi-styles@^3.2.0, ansi-styles@^3.2.1:
   dependencies:
     color-convert "^1.9.0"
 
+any-observable@^0.3.0:
```

Comment:
noticed this and wanted to call it out, should this be here?

Reply:
this is _probably_ from the `lint-staged` update here https://github.com/meetup/meetup-web-platform/pull/599/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R71 - i believe that uses rxjs behind the scenes, but as long as our `package.json`s don't specify rxjs, we should be fine. I can try rebuilding `yarn.lock` to see if it makes a difference

Reply:
yeah, just running `yarn whyr rxjs` will show you where it's being hoisted from and it is lint-staged. no need to worry about these